src/client/gfs_client.h

```diff
@@ -0,0 +1,44 @@
+#ifndef CLIENT_GFS_CLIENT_H
+#define CLIENT_GFS_CLIENT_H
+
+#include "google/protobuf/stubs/status.h"
+#include "google/protobuf/stubs/statusor.h"
+
+namespace gfs {
+namespace client {
+
+// Use the Status and Statusor objects provided by protobuf
+using google::protobuf::util::Status;
+using google::protobuf::util::StatusOr;
+
+/* Data type for the return of read operation. This includes how many bytes have been
+ * read and a pointer of the buffer */
+struct Data {
+   int bytesRead;
+   void* buffer; 
+};
+
+/* Provide an interface function to let user to initialize client and configure
+ * parameters such as master's url. A user should call this function before
+ * performing any of the following functions as initializations are needed */
+void configClient(const char* masterUrl);
+
+Status create(const char* pathname);
```

Comment:
do we still need this, if we have an `open` with `create` mode?

Reply:
@xicheng87  you skipped this

Reply:
This was also addressed - create() was removed from the last change.